rest-service/manager_rest/deployment_update/manager.py

```diff
@@ -325,7 +336,15 @@ def commit_deployment_update(self,
 
         rm.create_resource_labels(models.DeploymentLabel, deployment,
                                   labels_to_create)
-
+        if parents_labels:
```

Comment:
What this part is for?

Reply:
This is in order to increase the deployment counters if `csys-obj-parent` is added as part of deployment update to that deployment so that we can increase the number of deployment for this parent since it has now extra deployment attached to it